receiver/googlecloudmonitoringreceiver/README.md

```diff
@@ -41,8 +41,10 @@ receivers:
 
 Each single metric can have the following configuration:
 
-- `metric_name` (Required): The specific metric name to collect.
+- `metric_name` (Optional): The specific metric name to collect.
+- `monitoring_filter` (Optional): The [monitoring filter](https://cloud.google.com/monitoring/api/v3/filters) to filter metrics.
 
+One of `metric_name` and `monitoring_filter` MUST be specified, but should not be specified at the same time.
```

Comment:
Based on https://cloud.google.com/monitoring/api/v3/filters, if we are applying these to only to ListMetricDescriptor, there are a lot of restrictions on what you can filter by that might confuse users.  They can't use resource or group selectors, and presumably can't filter on labels: `metric.labels.instance_name = monitoring.regex.full_match("gke-(hipster|nginx).*")`.

One solution would be to make this a `metric_descriptor_filter` config option, so that it is clear we are filtering metric descriptors (and we should also document that there are restrictions on that filter).  That way we could later add a separate `time_series_filter` that we add to actual ListTimeSeries queries if we want.